frameworks/JavaScript/nodejs/app.js

```diff
@@ -0,0 +1,16 @@
+var cluster = require('cluster');
+var numCPUs = require('os').cpus().length;
+
+if (cluster.isMaster) {
+  // Fork workers.
+  for (var i = 0; i < numCPUs; i++) {
+    cluster.fork();
+  }
+
+  cluster.on('exit', function (worker, code, signal) {
+    console.log('worker ' + worker.pid + ' died');
```

Comment:
I've only seen this happen when an `err` is thrown in a database test. Otherwise this should never go off, it's more of a dev-helper as that log will be printed at the end of the stack trace that details why e.g. ERR_CONNECTION_REFUSED, the worker/thread died.